functions.php

```diff
@@ -172,10 +172,10 @@ function twentytwenty_menus() {
 
         // Register menus
         $locations = array(
-            'footer-menu' 		=> __( 'Footer Menu', 'twentytwenty' ),
-            'main-menu' 		=> __( 'Main Menu', 'twentytwenty' ),
-			'shortcuts-menu' 	=> __( 'Shortcuts Menu', 'twentytwenty' ),
-            'social-menu' 		=> __( 'Social Menu', 'twentytwenty' ),
+            'footer-menu'       => __( 'Footer Menu', 'twentytwenty' ),
+            'main-menu'         => __( 'Main Menu', 'twentytwenty' ),
+			'shortcuts-menu'    => __( 'Shortcuts Menu', 'twentytwenty' ),
```

Comment:
The shortcuts-menu has too many tabs.

Reply:
Good catch. Must be a mixed tabs/spaces issue. The fix is on the way 🙂